test/suite/wttest.py

```diff
@@ -752,10 +752,10 @@ def tearDown(self, dueToRetry=False):
 
         # Download the files from the S3 bucket for tiered tests if the test fails or preserve is
         # turned on.
-        if hasattr(self, 'ss_name') and self.ss_name == 's3_store' and not self.skipped and \
+        if hasattr(self, 'ss_name') and not self.skipped and \
             (not passed or WiredTigerTestCase._preserveFiles):
                 self.download_objects(self.bucket, self.bucket_prefix)
-                self.pr('downloading s3 files')
+                self.pr('downloading object files')
```

Comment:
I realise you didn't change it, but can you please move the print above the `download_objects` call? That way if it goes wrong we know what it tried to do, and the tense isn't incorrect when it finishes.

Reply:
done! [22b35ac](https://github.com/wiredtiger/wiredtiger/pull/8953/commits/22b35acd1575bfed2e228baece7e3d1775026f83)